python/python/glide/async_commands/cluster_commands.py

```diff
@@ -514,3 +514,40 @@ async def flushall(
             TClusterResponse[TOK],
             await self._execute_command(RequestType.FlushAll, args, route),
         )
+
+    async def lolwut(
+        self,
+        version: Optional[int] = None,
+        parameters: Optional[List[int]] = None,
+        route: Optional[Route] = None,
+    ) -> str:
```

Comment:
In case of multi-node route, the client receives a map. Maybe this fix help you to eliminate `mypy` errors. Update tests accrodingly
```suggestion
    ) -> TClusterResponse[str]:
```